<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="3" ignored="0" total="8" passed="5">
  <reporter-output>
  </reporter-output>
  <suite name="Guru99Project" duration-ms="68493" started-at="2020-08-17T14:59:48 EDT" finished-at="2020-08-17T15:00:57 EDT">
    <groups>
    </groups>
    <test name="Banking Project" duration-ms="68493" started-at="2020-08-17T14:59:48 EDT" finished-at="2020-08-17T15:00:57 EDT">
      <class name="com.guru99project.testCases.TC_LoginTest_001">
        <test-method status="PASS" signature="setUp(java.lang.String)[pri:0, instance:com.guru99project.testCases.TC_LoginTest_001@57e1b0c]" name="setUp" is-config="true" duration-ms="5406" started-at="2020-08-17T14:59:48 EDT" finished-at="2020-08-17T14:59:54 EDT">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="loginTest()[pri:0, instance:com.guru99project.testCases.TC_LoginTest_001@57e1b0c]" name="loginTest" duration-ms="869" started-at="2020-08-17T14:59:54 EDT" finished-at="2020-08-17T14:59:55 EDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.guru99project.testCases.TC_LoginTest_001@57e1b0c]" name="tearDown" is-config="true" duration-ms="625" started-at="2020-08-17T14:59:55 EDT" finished-at="2020-08-17T14:59:55 EDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.guru99project.testCases.TC_LoginTest_001 -->
      <class name="com.guru99project.testCases.TC_LoginDDT_002">
        <test-method status="PASS" signature="setUp(java.lang.String)[pri:0, instance:com.guru99project.testCases.TC_LoginDDT_002@4232c52b]" name="setUp" is-config="true" duration-ms="4315" started-at="2020-08-17T14:59:55 EDT" finished-at="2020-08-17T15:00:00 EDT">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="loginDDT(java.lang.String, java.lang.String)[pri:0, instance:com.guru99project.testCases.TC_LoginDDT_002@4232c52b]" name="loginDDT" duration-ms="10817" started-at="2020-08-17T15:00:01 EDT" data-provider="LoginData" finished-at="2020-08-17T15:00:12 EDT">
          <params>
            <param index="0">
              <value>
                <![CDATA[mngr277552]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[ubutUmU]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginDDT -->
        <test-method status="FAIL" signature="loginDDT(java.lang.String, java.lang.String)[pri:0, instance:com.guru99project.testCases.TC_LoginDDT_002@4232c52b]" name="loginDDT" duration-ms="3634" started-at="2020-08-17T15:00:12 EDT" data-provider="LoginData" finished-at="2020-08-17T15:00:15 EDT">
          <params>
            <param index="0">
              <value>
                <![CDATA[mngr277550]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[ubutUmU]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertTrue(Assert.java:43)
at org.testng.Assert.assertTrue(Assert.java:53)
at com.guru99project.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:28)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginDDT -->
        <test-method status="PASS" signature="loginDDT(java.lang.String, java.lang.String)[pri:0, instance:com.guru99project.testCases.TC_LoginDDT_002@4232c52b]" name="loginDDT" duration-ms="10075" started-at="2020-08-17T15:00:15 EDT" data-provider="LoginData" finished-at="2020-08-17T15:00:25 EDT">
          <params>
            <param index="0">
              <value>
                <![CDATA[mngr277552]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[ubutumU]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginDDT -->
        <test-method status="FAIL" signature="loginDDT(java.lang.String, java.lang.String)[pri:0, instance:com.guru99project.testCases.TC_LoginDDT_002@4232c52b]" name="loginDDT" duration-ms="3745" started-at="2020-08-17T15:00:25 EDT" data-provider="LoginData" finished-at="2020-08-17T15:00:29 EDT">
          <params>
            <param index="0">
              <value>
                <![CDATA[mngr270552]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[ubutumU]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertTrue(Assert.java:43)
at org.testng.Assert.assertTrue(Assert.java:53)
at com.guru99project.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:28)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginDDT -->
        <test-method status="FAIL" signature="loginDDT(java.lang.String, java.lang.String)[pri:0, instance:com.guru99project.testCases.TC_LoginDDT_002@4232c52b]" name="loginDDT" duration-ms="3610" started-at="2020-08-17T15:00:29 EDT" data-provider="LoginData" finished-at="2020-08-17T15:00:33 EDT">
          <params>
            <param index="0">
              <value>
                <![CDATA[mngr270552]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[ubutUmU]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.failNotEquals(Assert.java:969)
at org.testng.Assert.assertTrue(Assert.java:43)
at org.testng.Assert.assertTrue(Assert.java:53)
at com.guru99project.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:28)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginDDT -->
        <test-method status="PASS" signature="loginDDT(java.lang.String, java.lang.String)[pri:0, instance:com.guru99project.testCases.TC_LoginDDT_002@4232c52b]" name="loginDDT" duration-ms="9978" started-at="2020-08-17T15:00:33 EDT" data-provider="LoginData" finished-at="2020-08-17T15:00:43 EDT">
          <params>
            <param index="0">
              <value>
                <![CDATA[mngr277552]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[ubutUmU]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginDDT -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.guru99project.testCases.TC_LoginDDT_002@4232c52b]" name="tearDown" is-config="true" duration-ms="665" started-at="2020-08-17T15:00:43 EDT" finished-at="2020-08-17T15:00:43 EDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.guru99project.testCases.TC_LoginDDT_002 -->
      <class name="com.guru99project.testCases.TC_AddCustomerTest_003">
        <test-method status="PASS" signature="setUp(java.lang.String)[pri:0, instance:com.guru99project.testCases.TC_AddCustomerTest_003@1877ab81]" name="setUp" is-config="true" duration-ms="4328" started-at="2020-08-17T15:00:43 EDT" finished-at="2020-08-17T15:00:48 EDT">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="addNewCustomer()[pri:0, instance:com.guru99project.testCases.TC_AddCustomerTest_003@1877ab81]" name="addNewCustomer" duration-ms="8362" started-at="2020-08-17T15:00:48 EDT" finished-at="2020-08-17T15:00:56 EDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addNewCustomer -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.guru99project.testCases.TC_AddCustomerTest_003@1877ab81]" name="tearDown" is-config="true" duration-ms="654" started-at="2020-08-17T15:00:56 EDT" finished-at="2020-08-17T15:00:57 EDT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.guru99project.testCases.TC_AddCustomerTest_003 -->
    </test> <!-- Banking Project -->
  </suite> <!-- Guru99Project -->
</testng-results>
